### ISSUE TITLE
Optimizer settings for DONUT pre-training on Synthdog

### ISSUE THREAD

**Issue by Veason-silverbullet** [clovaai/donut#34]
Hi, @gwkrsrch ,

Many thanks for your effort to unblock the issues! I am trying to reproduce pre-training DONUT-proto on Synthdog, but I cannot get reasonable results. Could you please reveal the optimizer setting (i.e., settings of torch.optim.Adam andscheduler ) of DONUT-proto pretraining? It would be a great help to reproduce the pre-training!

**Comment by gwkrsrch** [comment]
Hi @Veason-silverbullet ,

The architecture settings are available at [config.json](https://huggingface.co/naver-clova-ix/donut-proto/blob/official/config.json) and more information is available in the manuscript (https://arxiv.org/abs/2111.15664), but I believe you have already checked it :) `donut-proto` shares most settings with `donut-base`, but it used 8e-5 as a constant rate. I wonder if your model training converged or not. Note that the `donut-proto` is trained with 5 GPU days.
Plus, note that `donut-proto` used "<s_synthdog>" as a task start token for **all** SynthDoG-generated data, as it can be checked at https://huggingface.co/naver-clova-ix/donut-proto/blob/official/added_tokens.json#L1. You can manually set the token with:
- https://github.com/clovaai/donut/blob/1.0.7/train.py#L83-L85

Hope this helps :) Please let me know if you are still confused. Feel free to reopen this or open another issue if you have anything new for sharing.

**Comment by Veason-silverbullet** [comment]
Hi @gwkrsrch ,
Yes, the model I trained may not converge. May I know the batch size and final training loss of DONUT-proto pretraining, so that I can know if my experimental model converged?

**Comment by gwkrsrch** [comment]
Yes, it was 1 sample per V100, and the normalized edit distance for the dev set was 0.0045.
